content/security/for-admins/single-sign-on/configure/_index.md

```diff
@@ -30,7 +39,7 @@ Follow the steps on this page to configure SSO for your organization or company.
 {{< /tab >}}
 {{< /tabs >}}
 
-## Step two: Create an SSO connection
+## Create an SSO connection in Docker
 
 {{< tabs >}}
```

Comment:
Agree these can use an update. Would actually like to do this in prod but would need a domain verified on the test org to do so. I'll see if this can be a fast-follow!